index.html

```diff
@@ -1,133 +1,111 @@
-       <div class="container"> 
-        <div>
-         <h2 class="featured">Featured Speackers</h2>
-        </div>
-        <div class="speackersContainer">
-            <div class="speacker">
-                <div >
-                <img class="speackerimg" src="./img/speackerone.png" alt="">
-                </div>
-                <div>
-                    <h2>Elon Musk</h2>
-                    <h4>Chief Engineer, Space X</h4>
-                    <p>Elon Reeve Musk FRS is an entrepreneur and business magnate. He is the founder, <br> 
-                        CEO and Chief Engineer at SpaceX; early stage investor</p>
-                </div>
-            </div>
-            <div class="speacker">
-                <div>
-                <img class="speackerimg" src="./img/speackertwo.jpg" alt="">
-                </div>
-                <div>
-                    <h2>Mark Zuckerberg</h2>
-                    <h4>Co-founder, Chairperson and CEO Facebook</h4>
-                    <P>Mark Elliot Zuckerberg is a media magnate, internet entrepreneur, <br> 
-                        and philanthropist. He is known for co-founding Facebook</P>
-                </div>
-            </div>
-            <div class="speacker">
-                <div>
-                <img class="speackerimg" src="./img/speackerthree.jpg" alt="">
-                </div>
-                <div>
-                    <h2>Amina Mohammed</h2>
-                    <h4>Deputy Secretary-General United Nations</h4>
-                    <p>Ms. Amina J. Mohammed is the Deputy Secretary-General of the United Nations <br> 
-                        and Chair of the United Nations Sustainable Development Group.</p>
-                </div>
-            </div>
-            <div class="speacker">
-                <div>
-                <img class="speackerimg" src="./img/speackerfour.jpg" alt="">
-                </div>
-                <div>
-                    <h2>Cher Wang</h2>
-                    <h4>Co-founder and Chairperson, HTC corporation</h4>
-                    <p>Cher Wang is an entrepreneur and philanthropist. As co-founder and chairperson <br>
-                         (since 2007) of HTC Corporation and integrated chipset maker </p>
-                </div>
-            </div>
-            <div class="speacker">
-                <div> 
-                <img class="speackerimg" src="./img/speackerfive.jpg" alt="">
-                </div>
-                <div>
-                    <h2>Reed Hastings</h2>
-                    <h4>CEO Netflix</h4>
-                    <p>Is a businessman. He is the co-founder, chairman, and co-chief executive officer (CEO) of Netflix.</p>
-                </div>
+        <div class="container">
+            <div>
+                <h2 class="featured">Featured Speackers</h2>
             </div>
+            <div class="speackersContainer" id="speackersContainer"></div>
         </div>
-
-      </div>
     </section>
```

Comment:
- [ ] On desktop view the featured speaker's section should be arranged in a column but on your page, it is displayed in a row as shown below.
Your design👇🏽
![Capture5](https://user-images.githubusercontent.com/61285247/133879130-c23f1e95-7c32-4b90-a726-9e5645182ca8.PNG)
Please kindly fix that.

- [ ] I can see that the whole page is overflowing. Please kindly fix that.